packages/tadviewer/src/components/GridPane.tsx

```diff
@@ -105,28 +103,52 @@ const GridPaneInternal: React.FunctionComponent<GridPaneProps> = ({
   );
   const sortKey = viewParams.sortKey;
 
+  const onGridSelectionChange = React.useCallback(
+    (anchor: Cell, focus: Cell, columns: string[], items: any[][]) => {
+      const appState = mutableGet(stateRef);
+      const { viewState } = appState;
+      if (onSelectionChange) {
+        const columnData: ColumnData[] = [];
+        columns.map((column) => {
+          columnData.push({
```

Comment:
It's awkward to mix `map` and `push`. Perhaps you can just return each ColumnData from the map lambda, and assign the result of the map to `columnData` array?

Reply:
Fixed